reference-architectures/cloud_deploy_flow/main.tf

```diff
@@ -0,0 +1,127 @@
+# Configure the Google Cloud provider
+terraform {
```

Comment:
Set version constraints for terraform and providers here. Considering that you have a dependency lock file, constraints can be relatively loose.